buildpack.md

```diff
@@ -266,12 +266,12 @@ If a cache is provided, the lifecycle:
 
 Before the next re-build:
 - The lifecycle MUST do both or neither of the following:
-  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `launch`, `build`, and `cache` booleans.
+  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `[types]` table.
   - Restore layer contents to the `<layers>/<layer>/` directory.
 
 #### Ignored Layers
 
-Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
+Layers marked `launch = false`, `build = false`, and `cache = false` (or when the `[types]` table does not exist) behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
```

Comment:
Maybe we can remove this parenthetical and add a someone near [here](https://github.com/buildpacks/spec/pull/209/commits/6f968784d15143d02588a0b2dbbd5f1c92ce1dbd#diff-d5ffdcac80bcacec34b3a8a1ba9da8efbbc4df5ad118ec581201736330f73183R199) along the lines of: "The lifecycle MUST treat a layer with unset `types` as a `launch = false`, `build = false`, `cache = false` layer."

Reply:
Please see https://github.com/buildpacks/spec/pull/209/commits/f2eb2af7d05dff04d6570e2057321c45adca96ac.